### PR TITLE
HomeWork

### PR DESCRIPTION
**Pull request by MaxKharitonov**

### CODE REVIEW

File: HomeWork [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+public class lesson2 {
+    public static void main(String[] args) {
+    falseTrue();
```
Comment (MaxKharitonov) [inline HomeWork:4]: boolean x = falseTrue();
System.out.println(x);

File: HomeWork [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    polOtr();
+    pOTF();
```
Comment (MaxKharitonov) [inline HomeWork:6]: boolean y = pOTF();
System.out.println(y);

File: HomeWork [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    strInt();
+    year();
```
Comment (MaxKharitonov) [inline HomeWork:8]: boolean z = year();
System.out.println(z);

**Review by MaxKharitonov — COMMENTED** [review]
.

**Review by emgusev — APPROVED** [review]

**Pull request closed without merging**